features/weed_reported_location.feature

```diff
@@ -0,0 +1,68 @@
+The synchroniser reads the CAMS feature geolocation and compares it to the CAMS iNaturalist location fields to determine 
+whether the location has been updated by the user. If it has, the iNaturalist location will not override the CAMS geolocation.
+
+Rule: Whenever the iNaturalist location is updated, both the CAMS feature geolocation and the iNaturalist location fields are updated
+    Example: The CAMS iNaturalist location is set to the iNaturalist location for new observations
+        Given iNaturalist has a new observation at 'latitude': -41.2920665997, 'longitude': 174.7628175
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.2920665997, 'longitude': 174.7628175
+
+    Example: The CAMS iNaturalist location is updated when the iNaturalist location is changed
+        Given iNaturalist has an existing observation at 'latitude': -41.2920665997, 'longitude': 174.7628175
+        And that observation has been synced
+        And the geolocation is updated to 'latitude': -41.1234567, 'longitude': 174.7654321
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.1234567, 'longitude': 174.7654321
+    
+    Example: For an unchanged, existing CAMS feature, the CAMS geolocation and iNaturalist location are updated when the original iNaturalist location is changed
+        Given iNaturalist has an existing observation at 'latitude': -41.291111, 'longitude': 174.761111
+        And that observation has been synced
+        And the geolocation is updated to 'latitude': -41.292222, 'longitude': 174.762222
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.292222, 'longitude': 174.762222
+        And the CAMS geolocation is set to 'latitude': -41.292222, 'longitude': 174.762222
+
+    Example: For a changed CAMS feature, the CAMS geolocation and iNaturalist location are updated when the iNaturalist location is changed
+        Given iNaturalist has an existing observation at 'latitude': -41.291111, 'longitude': 174.761111
+        And that observation has been synced
+        And the CAMS geolocation has been manually updated to 'latitude': -41.123666, 'longitude': 174.123666
+        And the geolocation is updated to 'latitude': -41.292222, 'longitude': 174.762222
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.292222, 'longitude': 174.762222
+        And the CAMS geolocation is set to 'latitude': -41.292222, 'longitude': 174.762222
+
+Rule: After the CAMS user updates the geolocation, it is not overwritten unless the iNaturalist location is subsequently updated
+    Example: The CAMS geolocation is not updated if the CAMS iNaturalist location is the same as the iNaturalist location       
```

Comment:
I don't think this is the right description. Shouldn't it be something like:
On an update to the iNaturalist observation other than a location update, the CAMS geolocation is not updated if the CAMS iNaturalist location differs from the CAMS geolocation

Reply:
Or maybe the If bit should come first! God, it's hard to make the titles consistent across the feature!!

Reply:
I think it's this: The CAMS geolocation is only updated if the iNaturalist location changes